cmd/readme_test/readme.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
The reason cmd/test/main.go will at some point become become a full fledged test application with the api becomes complete. While the readme test I would like keep it uncluttered just to test the `README.md` code snippet